aspnet/migration/rc1-to-rc2.rst

```diff
@@ -0,0 +1,352 @@
+Migrating from ASP.NET to ASP.NET Core
+======================================
+
+.. contents:: Sections:
+  :local:
+  :depth: 1  
+
+Overview
+--------
+
+This migration guide covers migrating an ASP.NET 5 RC1 application to ASP.NET Core.
+
+ASP.NET 5 RC1 apps were based on the .NET Execution Environment (DNX) and made use of DNX specific features. ASP.NET Core RC2 is based on .NET Core, so you must first migrate your application to the new .NET Core project model. For information on migrating from DNX to .NET Core CLI see <need link>.
+
+For information regarding a complete list of breaking changes in RC2 for ASP.NET Core, see the `ASP.NET Core Announcements page <https://github.com/aspnet/announcements/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0.0-rc2>`_
+
+Creating your web application host
+----------------------------------
+
+Since ASP.NET Core apps are just console apps, you must define an entry point for your application in ``Program.Main`` that sets up a web host, then tells it to start listening. Below is an example of the startup code for the built-in `Web Application` template in Visual Studio.
+
+.. code-block:: c#
+
+  public class Program
+  {
+    public static void Main(string[] args)
+    {
+        var host = new WebHostBuilder()
+            .UseKestrel()
+            .UseContentRoot(Directory.GetCurrentDirectory())
+            .UseDefaultHostingConfiguration(args)
+            .UseIISIntegration()
+            .UseStartup<Startup>()
+            .Build();
+
+        host.Run();
+    }
+  }
+
+
+Configure the webroot and port in the ``hosting.json`` file. The web root of your application is now defined when setting up the web host and defaults to ``wwwroot``.
```

Comment:
Ahh, didn't see that you explained this later on in the document. Though I still think the phrasing here is a little misleading. It sounds like you can simply create a `hosting.json` with the configuration you want and it will be automatically read by the web host which is not true.